op-e2e/e2eutils/disputegame/game_helper.go

```diff
@@ -287,7 +287,7 @@ func (g *FaultGameHelper) ChallengeRootClaim(ctx context.Context, performMove fu
 	g.LogGameData(ctx)
 
 	// It's on us to call step if we want to win but shouldn't be possible
-	attemptStep(maxDepth)
+	attemptStep(int64(maxDepth))
```

Comment:
I think this is a true positive, but I'm not sure how to address it since I'm not familiar with this part of the codebase. Why is `Stepper` treating a depth value as a claim index. Is that intentional? Should `parentClaimIdx` and similar variables be changed to type `Depth`?

Reply:
It is indeed a bit quirky. It's working on the basis that there is just a single "chain" of claims in the game so the claim index and depth are always equal. I'd stick with casting for the purposes of this PR, but we're moving things over to use the `ClaimHelper` type to reference a specific claim which makes this kind of logic a lot less confusing. The `Mover` has already been updated to take a `ClaimHelper` but `Stepper` hasn't yet and probably should be.  Probably better to change it in a separate PR though.